db/migrate/20181122111014_convert_timestamp_without_tz_to_timestamp_with_tz.rb

```diff
@@ -0,0 +1,51 @@
+class ConvertTimestampWithoutTzToTimestampWithTz < ActiveRecord::Migration
+  def up
+
+    model_names = ['address', 'auth_token', 'beneficiary', 'booking_type', 'box',
+    'braintree_transaction', 'cancellation_reason', 'contact', 'country',
+    'crossroads_transport', 'delivery', 'district', 'donor_condition', 'gogovan_order',
+    'gogovan_transport', 'goodcity_request', 'holiday', 'identity_type', 'image',
+    'inventory_number', 'item', 'location', 'message', 'offer', 'order', 'orders_package',
+    'orders_purpose', 'order_transport', 'organisation', 'organisations_user',
+    'organisation_type', 'package_categories_package_type', 'package_category',
+    'package', 'packages_location', 'package_type', 'pallet', 'permission', 'purpose',
+    'rejection_reason', 'role_permission', 'role', 'schedule', 'stockit_activity',
+    'stockit_contact', 'stockit_local_order', 'stockit_organisation', 'subpackage_type',
+    'subscription', 'territory', 'timeslot', 'user', 'user_role', 'version'].freeze
+
```

Comment:
If these model_names are same, is there any way to write this array just once? 
thoughts @patrixr ?

Reply:
True, could do that, or extract them from postgres. But it's a data migration code which will run just once, so I'm okay with this :)

Reply:
cool 👍

Reply:
Good thinking Abhinav, I had the same thought but hadn't followed through with it. I'll create a constant at the top of the migration file.